src/core/public/chrome/chrome_service.tsx

```diff
@@ -496,3 +533,25 @@ export interface InternalChromeStart extends ChromeStart {
    */
   getHeaderComponent(): JSX.Element;
 }
+
+export enum GROUP_TYPE {
+  SYSTEM_GROUP = 'SYSTEM_GROUP',
+  USE_CASE_GROUP = 'USE_CASE_GROUP',
+}
+
+/** @public */
+export interface ChromeNavGroup {
+  id: string;
+  title: string;
+  description: string;
+  order?: number;
+  icon?: EuiIconType;
+  type: GROUP_TYPE; // group with type of GROUP_TYPE.SYSTEM_GROUP will always display before USE_CASE_GROUP
```

Comment:
can this accomplished by order if it only impacts the display order

Reply:
I agree with @Hailong-am; if the purpose of `type` is to display something higher, we could reserve `order` of negative for OSD et al. and positives for others.

Reply:
My fault, not only the order. It indicates if the group can be picked up as a use case in workspace creation page. Let me add more comment on it.